contrib/collectd/patches/disable_fqdn_lookup.patch

```diff
@@ -0,0 +1,11 @@
+--- a/src/collectd.conf.in	09:33:14.960686846 +0000
++++ a/src/collectd.conf.in	2024-07-24 09:33:21.835667786 +0000
+@@ -11,7 +11,7 @@
+ ##############################################################################
+
+ #Hostname    "localhost"
+-#FQDNLookup   true
++FQDNLookup   false
```

Comment:
why default this to false?

Reply:
So this may just be a thing on my machine (thinkpad with the full gnome chimera image installed), but when it defaults to true the service fails to start because my laptop does not have a fqdn and the lookup fails.  Someone else noted this [here](https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=779483), along with some discussion about adding a new default to cover this case, but they didn't add that new default.  It's possible my system is misconfigured.

Reply:
ah yea

seems ok then